functions/Export-SqlUser.ps1

```diff
@@ -96,7 +96,8 @@ https://dbatools.io/Export-SqlUser
 		[System.Management.Automation.PSCredential]$SqlCredential,
 		[Alias("NoOverwrite")]
 		[switch]$NoClobber,
-		[switch]$Append
+		[switch]$Append,
+		[switch]$IncludeSystemDatabases = $false
```

Comment:
Oh, we'll also need the Help parameter to be added - currently, the Pester tests that check to ensure all params have documentation are failing.